options.js

```diff
@@ -18,4 +19,29 @@ function saveChanges(inputElement) {
     chrome.storage.local.set(storageObject);
 }
 
+function showTechnologyOptions(){
+    var technology = document.getElementById("technology");
+    var sapInput = document.getElementById("sap-input");
+    var tsProject = document.getElementById("ts-project-input");
+    var tsServerDescription = document.getElementById("ts-server-description");
+    var webServerDescription = document.getElementById("web-server-description");
+
+    if ( technology.value == 'dotnet')
+      {
+        sapInput.hidden = true ;
+        tsProject.hidden = true;
+        tsServerDescription.hidden = true;
+        webServerDescription.hidden = false;
+      }     
+        else
+      {
+        sapInput.hidden = false ;
+        tsProject.hidden = false;
+        tsServerDescription.hidden = false;
+        webServerDescription.hidden = true;
+     }
+}
+
+document.getElementById("technology").addEventListener("change", showTechnologyOptions);
```

Comment:
Can we use the element's variable/`const` from above instead of repeating `document.getElementById("technology")`?